.gitlab-ci.yml

```diff
@@ -321,6 +321,14 @@ cargo-fmt:
   script:
     - cargo +nightly fmt --all -- --check
 
+cargo-clippy:
+  stage:                           test
+  <<:                              *docker-env
+  <<:                              *test-refs
+  script:
+    - SKIP_WASM_BUILD=1 env -u RUSTFLAGS cargo +nightly clippy
+  allow_failure:                   true
```

Comment:
Please don't allow it to fail. It only frustrates people that are trying to get their CI green while other PRs add regressions. If that stuff works it shouldn't be optional.

Reply:
Given we're focused on only correctness lints I think this is a reasonable decision. I'm not expecting many failing builds due to it.